packages/fxa-settings/src/components/PageSecondaryEmailAdd/index.tsx

```diff
@@ -17,6 +17,8 @@ export const PageSecondaryEmailAdd = (_: RouteComponentProps) => {
   const [email, setEmail] = useState<string>();
   const inputRef = useRef<HTMLInputElement>(null);
   const { l10n } = useLocalization();
+  composeAuthUiErrorTranslationID({ errno: 123, message: 'This is an error!'});
```

Comment:
I think removing this might've been missed when you were testing? 👀

Reply:
drat definitely 😂 cleaning up now